web/src/components/software/ProductSelectionPage.jsx

```diff
@@ -88,16 +90,27 @@ function ProductSelectionPage() {
 
   return (
     <>
-      <Title>Product selection</Title>
+      {/* TRANSLATORS: page header */}
+      <Title>{_("Product selection")}</Title>
       <PageIcon><Icon name="home_storage" /></PageIcon>
       <MainActions>
         <Button isLarge variant="primary" form="product-selector" type="submit">
-          Select
+          {/* TRANSLATORS: button label */}
+          {_("Select")}
         </Button>
       </MainActions>
 
       <Form id="product-selector" onSubmit={accept}>
-        <FormGroup isStack label={`Choose a product (${products.length} available)`} role="radiogroup">
+        <FormGroup
+          isStack
+          label={
+            format(
+              n_("Choose a product (%d available)", "Choose a product (%d available)", products.length),
```

Comment:
Here I'm not sure if we really need to display the number of available products.

- There are usually just few products
- The number does not provide any meaningful information for the user, you can always choose *only one* product, it does not matter how many products are available

So I'd rather make it easier and use just simple `_("Choose a product")` here.

Opinions?

Reply:
Yes, I agree, I would get rid of the number of available products.